extras/userdoc/md/documentation/an-introduction-to-sli.md

```diff
@@ -10,7 +10,7 @@ at the command prompt. You should then see something like this:
     gewaltig@jasmin-vm:~$ nest
                 -- N E S T 2 beta --
 
-      Copyright 1995-2009 The NEST Initiative
```

Comment:
This is not a copyright notice, but part of example output (which prints a copyright notice). The proper thing to do here would be to run the example and paste the output with an up-to-date nest. Otherwise, the text needs to stay as it is for consistency.